src/main/java/it/gov/pagopa/exception/RestExceptionHandler.java

```diff
@@ -26,14 +26,12 @@ public ResponseEntity<Object> handleBadRequest(final DataIntegrityViolationExcep
         return handleExceptionInternal(ex, bodyOfResponse, new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
     }
 
-    @Override
```

Comment:
have you removed this Override because it will override nothing?
if yes, this is an error! this override was performed to override default behavior to handle HttpMessageNotReadableException exception, this way you are suppressing it